test/unit/instrumentation/connect.test.js

```diff
@@ -5,233 +5,211 @@
 
 /* eslint-disable strict */
 
-const tap = require('tap')
+const test = require('node:test')
+const assert = require('node:assert')
 const helper = require('../../lib/agent_helper')
 const WebShim = require('../../../lib/shim/webframework-shim')
 
 function nextulator(req, res, next) {
   return next()
 }
 
-tap.test('an instrumented Connect stack', function (t) {
-  t.autoend()
+test("shouldn't cause bootstrapping to fail", async function (t) {
+  // testing some stuff further down that needs to be non-strict
+  'use strict'
+  const agent = helper.loadMockedAgent()
+  const shim = new WebShim(agent, 'connect')
+  const initialize = require('../../../lib/instrumentation/connect')
 
-  t.test("shouldn't cause bootstrapping to fail", function (t) {
-    // testing some stuff further down that needs to be non-strict
-    'use strict'
-
-    t.autoend()
-    let agent
-    let initialize
-    let shim
-
-    t.before(function () {
-      agent = helper.loadMockedAgent()
-      shim = new WebShim(agent, 'connect')
-      initialize = require('../../../lib/instrumentation/connect')
-    })
-
-    t.teardown(function () {
-      helper.unloadAgent(agent)
-    })
+  t.after(function () {
+    helper.unloadAgent(agent)
+  })
```

Comment:
i didn't add beforeEach because it was before and teardown.  and realized before is useless so i just hoisted the before work within the parent test and added a teardown

Reply:
i'll just fix this